types/index.d.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference types='node' />
+
+import {
+  FastifyPluginCallback,
+} from 'fastify';
+
+declare module 'fastify' {
+  interface FastifyContextConfig {
+    throttle?: fastifyThrottle.FastifyThrottleOptions;
+  }
+}
+
+type FastifyThrottle = FastifyPluginCallback<fastifyThrottle.FastifyThrottleOptions>;
+
+declare namespace fastifyThrottle {
+
+  export interface FastifyThrottleOptions {
+    bps: number | ((elapsedTime: number, bytes: number) => number)
```

Comment:
I would suggest using full names instead of an abbreviation that is not self-explanatory

Reply:
renamed to bytesPerSecond